desktop/src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java

```diff
@@ -463,6 +511,30 @@ private void onSendBtc() {
         }
     }
 
+    private byte[] getOpReturnData(String preImageAsString) {
+        byte[] opReturnData;
+        try {
+            // If preImage is hex encoded we use it directly
+            opReturnData = Hex.decode(preImageAsString);
+        } catch (Throwable ignore) {
+            opReturnData = preImageAsString.getBytes(Charsets.UTF_8);
+        }
+
+        // If too long for OpReturn we hash it
+        if (opReturnData.length > 80) {
```

Comment:
Is 80 bytes going to be relayed?

Reply:
https://www.talkcrypto.org/blog/2016/12/30/op_return-40-to-80-bytes/

https://www.oreilly.com/library/view/mastering-bitcoin/9781491902639/ch05.html

```
OP_RETURN was initially proposed with a limit of 80 bytes, but the limit was reduced to 40 bytes when the feature was released. In February 2015, in version 0.10 of Bitcoin Core, the limit was raised back to 80 bytes. Nodes may choose not to relay or mine OP_RETURN, or only relay and mine OP_RETURN containing less than 80 bytes of data.
```

Not 100% clear if there might be issues with nodes not relaying, but seems that 80 bytes is standard limit.